.storybook/docs-root.css

```diff
@@ -156,6 +156,16 @@
   border-radius: 5px 5px 0px 0px !important;
 }
 
+/* Reduce font size of CodeSandbox and Show Code button when zoomed or small window width*/
+/* https://github.com/microsoft/fluentui/issues/22764 */
+
+@media screen and (max-width: 380px) {
+  #docs-root .docblock-code-toggle,
+  .docs-story > a:last-child {
```

Comment:
I think the most scalable solution is to modify the codesandbox addon to add a class so that these overrides are more future proof. If we ever add a new link these styles will just break

Reply:
agree with @ling1726 , any kind of custom styles we provide agains private APIs ( most of storybook styling ) introduces very brittle code and additional overhead needed from our side.

Reply:
https://github.com/microsoft/fluentui-storybook-addons/pull/16

Reply:
how about we kick that cleanup out to another issue rather than blocking this fix. targeting the nth child link has already been done several times, why block my PR?

Reply:
can you add link to that PR to this change so it is tracked and as a followup remove it ?

Reply:
added https://github.com/microsoft/fluentui-storybook-addons/pull/16 to above PR description